sirplatform.h

```diff
@@ -124,22 +170,18 @@
 
 # define SIR_MAXHOST 256
 
-# if defined(__GLIBC__)
-#  if (__GLIBC__ >= 2 && __GLIBC_MINOR__ > 19) || \
-     ((__GLIBC__ == 2 && __GLIBC_MINOR__ <= 19) && defined(_BSD_SOURCE))
-#   define __HAVE_UNISTD_READLINK__
-#  endif
-# endif
-
 # if !defined(__WIN__)
```

Comment:
I swear I specifically recall merging this in. I have no idea what happened there.

Reply:
I kind of hate git - I think the issue is automatic resolutions (git-rere) stuff, but I guess having Microsoft give us a free platform makes using it worth the pain.  So honestly it probably wasn't "you", really.